packs/one-shot-bestiary/darius.json

```diff
@@ -455,96 +455,80 @@
                     {
                         "domain": "all",
                         "key": "RollOption",
-                        "label": "Darius has an active mutagen effect",
+                        "label": "PF2E.NPCAbility.Darius.ActiveMutagen",
```

Comment:
I don't know that we even need a label, this ability is kind of weird I think.

Reply:
It is pretty weird. I thought about not having a custom label, but I'm not sure which state "Metabolize Mutagen" describes. Suggestions?

Reply:
I think we may want to just redo a lot of his automation. Maybe a custom effect for the mutagen for him to get him the right modifiers and all that.

Reply:
Moved to an effect.